.github/ISSUE_TEMPLATE/onboarding_argocd.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Onboard to Continuous Deployment
+about: Requesting application to be automatically deployed via ArgoCD to any managed OpenShift cluster
+title: ""
+labels: onboarding
+assignees: ""
+---
+
+### Requested actions
+
+Select all that apply:
+
+- [ ] Create a new ArgoCD project for existing team
+- [ ] Enlist an application to be deployed automatically via ArgoCD
+
+### Questionnaire
+
+#### Creating a new project
+
+1. **Team name**:
+   Please tell us your team name, which corresponds to [an already available user `Group`](https://github.com/operate-first/apps/tree/master/cluster-scope/base/groups).
+
+2. **Target cluster**:
+   Choose an environment from [this list](https://github.com/operate-first/apps/tree/master/cluster-scope/overlays)
+
+3. **Managed namespaces**:
+   The target namespaces that should be managed by ArgoCD. Preferably using a single (team name) prefix.
```

Comment:
Can we link to [here](https://github.com/operate-first/argocd-apps/blob/main/docs/onboard_team_to_argocd.md#additional-notes) to for further explanation of why a prefix is preferred.